Library/DiscUtils.Vhdx/LogEntry.cs

```diff
@@ -121,7 +121,7 @@ public static bool TryRead(Stream logStream, out LogEntry entry)
             byte[] logEntryBuffer = new byte[header.EntryLength];
             Array.Copy(sectorBuffer, logEntryBuffer, LogSectorSize);
 
-            StreamUtilities.ReadFully(logStream, logEntryBuffer, LogSectorSize, logEntryBuffer.Length - LogSectorSize);
+            StreamUtilities.ReadExact(logStream, logEntryBuffer, LogSectorSize, logEntryBuffer.Length - LogSectorSize);
```

Comment:
@zivillian, good catch, thanks. I went back looking for similar issues and found none.